api/views/views.py

```diff
@@ -0,0 +1,34 @@
+from flask_restplus import Resource
+from flask import Flask, request, jsonify
+
+from api.models.models import Users
+from main import api
+
+from api.utilities.schemas.user_schema import UserSchema
+from api.utilities.messages.success_messages import SUCCESS_MESSAGES
+from api.utilities.validators.user_validator import check_user
+
+
+@api.route('/registration')
+class Registration(Resource):
+    """
+    Resource class for peforming crud on the asset categories
+    """
+
+    def get(self):
+        return {'status': 'success'}, 200
```

Comment:
Why do you have a get request for registration? Shouldn't it return a 405 instead?